### ISSUE TITLE
OMIABIS ontology is empty

### ISSUE THREAD

**Issue by djvanenckevort** [OntoZoo/ontobee#79]
When I try to download the terms list for OMIABIS I get an empty Excel sheet, also when I try to browse to an individual term it is not found (e.g. http://purl.obolibrary.org/obo/OMIABIS_0001054).

**Comment by e4ong1031** [comment]
I have checked the ontology source file. There are three import files: http://purl.obolibrary.org/obo/omiabis/externalByHand.owl; http://purl.obolibrary.org/obo/omiabis/import_OBI_subset.owl; http://purl.obolibrary.org/obo/omiabis/import_OBO.owl.

These URLs return 404 status and do not map to ontology files. Ontobee will automatically download these import files. Merged the ontologies and then upload to the RDF database.

If the three import ontologies are not necessary to view OMIABIS, I can give OMIABIS a "do-not-merge" flag and bypass downloading these three files.
